source/sessions/driver-sessions.rst

```diff
@@ -830,6 +830,13 @@ of the cursor.  For language runtimes that provide the ability to attach finaliz
 that are run prior to garbage collection, the cursor class SHOULD return an implicit session
 to the pool in the finalizer if the cursor has not already been exhausted.
 
+If a driver supports process forking, the session pool needs to be cleared on
+one side of the forked processes (just like sockets need to reconnect).
+Drivers MUST provide a way to clear the session pool without sending
+``endSessions``.  Drivers MAY make this automatic when the process ID changes.
+If they do not, they MUST document how to clear the session pool wherever they
+document fork support.
```

Comment:
Let's name this API so drivers that provide it use a consistent name, and let's add a prose test to tests/README.

Reply:
We don't define any API for session pools and many (most?) drivers may not expose them to end-users.  E.g. the Perl driver provides a `reconnect` method on MongoClient that is documented as the thing to call after forking.  Inside that method, the driver reconnects sockets and now clears the pool.